examples/html-css/index.html

```diff
@@ -3,10 +3,15 @@
   <head>
     <meta charset="utf-8" />
     <title>Design System</title>
-    <link rel="stylesheet" href="../../packages/core/dist/index.css" />
+    <link rel="stylesheet" href="../../packages/design-system/dist/css/index.css" />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body class="ds-base">
+    <header class="ds-u-padding--3 ds-u-sm-padding--3 ds-u-display--block ds-u-fill--gray-light">
```

Comment:
I thought we decided on three different HTML examples inside `html-css`? So instead of defining a new HTML file just literally copying the 3 html examples into a folder called `html-css`

Reply:
Yes, we did, and yesterday I tried merging the examples and it looks pretty good. Check it out and see what you think. I have then updated the docs that references this example link.

Reply:
This styling seems super heavy handed
<img width="1008" alt="Screen Shot 2020-08-06 at 12 15 13 PM" src="https://user-images.githubusercontent.com/5424713/89555887-98884380-d7de-11ea-8a09-5b9360c14f2e.png">

Can we remove the gray fill and label each example like so?
<img width="1024" alt="Screen Shot 2020-08-06 at 12 15 41 PM" src="https://user-images.githubusercontent.com/5424713/89555933-a938b980-d7de-11ea-9f69-e4ccf1d0cdc5.png">

Reply:
That looks much better!